source/faq/faq.rst

```diff
@@ -356,93 +399,83 @@ map the input attributes to the corresponding ones into an output, and  finally
 I want to aggregate the attributes of multiple devices, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-First of all, you need to create a virtual device to aggregate all attributes, then you build a 
-data flow to map the attributes of each device to the virtual one. This can be done connecting a 
-‘change’ node after each input device to put the input values into an output, and finally 
-connecting all changes to the virtual device and assigning to it the output.
+First of all, you need to create a virtual device to aggregate all attributes,
+then you build a data flow to map the attributes of each device to the virtual
+one. This can be done connecting a ‘change’ node after each input device to put
+the input values into an output, and finally connecting all changes to the
+virtual device and assigning to it the output.
 
 .. image:: df_attributes_aggregation.gif
         :width: 95%
         :align: center
 
-It would be cool a WhatsApp node, is it in roadmap?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-It’s under analysis. We intend to support other notifications systems besides email, including 
-WhatsApp, Twitter and Telegram. If you also have interest, please help us to develop them.
-
-Applications
-------------
-.. _applications:
-
-What APIs are available for applications?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can check all available APIs in the `API Listing page <../apis.html>`_
-
-How can I use them?
-^^^^^^^^^^^^^^^^^^^
-
-First, you will need an access token, which can be retrieved sending a HTTP POST request to 
-``/auth`` endpoint with the following JSON content:
-
-.. code-block:: javascript
-
-	{  "username" : <>, "passwd" : <> }
+How can I add a new node type to its menu?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Obviously the values of each attribute should be correctly filled in. An example of such 
-request using ``curl`` would be:
+It's pretty easy, actually, although it needs a few commands in bash. To add a
+new node, you should send the following request:
 
 .. code-block:: bash
 
-	$ curl -X POST http://localhost:8000/auth -H 'Content-Type:application/json' \
-	$ -d '{"username" : "admin", "passwd" : "admin"}'
+    curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node
+    -H "content-type: application/json" -d '{"image": "mmagr/kelvin:latest",
+    "id":"kelvin"}'
 
-which gives us back:
+This will add a new node called 'kelvin' which is implemented by a docker image
```

Comment:
The issue of having to pull the image before adding it to the flow is solved, isn't it?

Reply:
@rascaraficci, did you still have to pull these images?

Reply:
dojot/flowbroker#19